Libraries/PushNotificationIOS/RCTPushNotificationManager.m

```diff
@@ -407,6 +408,30 @@ - (void)handleRegisterUserNotificationSettings:(NSNotification *)notification
   callback(@[formattedScheduledLocalNotifications]);
 }
 
+RCT_EXPORT_METHOD(removeAllDeliveredNotifications)
+{
+  if ([UNUserNotificationCenter class]) {
+    UNUserNotificationCenter *center = [UNUserNotificationCenter currentNotificationCenter];
+    [center removeAllDeliveredNotifications];
+  }
+}
+
+RCT_EXPORT_METHOD(removeDeliveredNotifications:(NSArray<NSString *> *)identifiers)
+{
+  if ([UNUserNotificationCenter class]) {
+    UNUserNotificationCenter *center = [UNUserNotificationCenter currentNotificationCenter];
+    [center removeDeliveredNotificationsWithIdentifiers:identifiers];
+  }
+}
+
+RCT_EXPORT_METHOD(getDeliveredNotifications:(RCTResponseSenderBlock)callback)
+{
+  if ([UNUserNotificationCenter class]) {
+    UNUserNotificationCenter *center = [UNUserNotificationCenter currentNotificationCenter];
+    [center getDeliveredNotificationsWithCompletionHandler:callback];
```

Comment:
This is not going to work. This is typed to be NSArray<UNNotification *>. which can't be send to JS just like that. You'll need to transform it to a dictionary first yourself.